cub/cub/device/dispatch/tuning/tuning_select_if.cuh

```diff
@@ -45,6 +45,8 @@
 #include <cub/util_math.cuh>
 #include <cub/util_type.cuh>
 
+#include <cuda/std/__algorithm_>
+
```

Comment:
It turns out this include broke a unit test :D Because now we have more `transform` overloads to find via ADL.